samphillips1879/Memory-Trunk
# Profile, Memory, Tip, and Perspective models created

## Description
This initializes the Profile, Memory, Tip, and Perspective models. All models are fully tested and are modularized by class.


## Related Ticket(s)
Fixed: #4 #7 #8 #9 


## Changes
4 models created with tests and linked to the admin page


## Expected Behavior
All four models can be instantiated from the admin page, all tests pass


## Steps to Test Solution
From the root of the Django project, run the following commands:
```
git fetch --all
git checkout origin/models
python manage.py migrate
python manage.py test
python manage.py runserver 8000
```
1. In your browser, navigate to `http://localhost:8000/admin/`
1. Make sure you can create new instances of each object


## Unit Testing

## Documentation


## Deploy Notes

Run `python manage.py migrate`
